tests/test-cases.js

```diff
@@ -1897,6 +1897,114 @@ var CUSTOM_TESTS = [
 			});
 		}
 	},
+	{
+		description: "Pause and resume works for chunks with NetworkStreamer (for Bug #736)",
+		disabled: !XHR_ENABLED,
+		timeout: 30000,
+		expected: ["SembCorp Industries Ltd", "Singapore", "SCIL.SI", "10%", "Yes", "0.30%"],
+		run: function(callback) {
+			var chunkNum = 0;
+			Papa.parse(BASE_PATH + "duplicate.csv", {
```

Comment:
Do we really need a new test file? 
Could you please use long-sample.csv? If you need some special character just edit the file and add them.